src/Control/Monad/Trans/OptionParser.hs

```diff
@@ -85,7 +86,7 @@ instance MonadIO m => MonadIO (OptionParserT o m) where
 mapOptionParserT :: (forall a. m a -> m' a)
                  -> OptionParserT o m b -> OptionParserT o m' b
 mapOptionParserT f = OptionParserT
```

Comment:
> `Move brackets to avoid $ `

That's a weird suggestion, I usually use `$` to avoid brackets! Is it possible to customize hlint to turn off certain kinds of suggestions if we don't agree with them? I agree with the 2 other hints, just not this one.